### PR TITLE
Ag 12360 improve demo example height

### PR DESCRIPTION
**Pull request by kylerphillips**
Provides a fix for smaller heights below 720px

### CODE REVIEW

File: documentation/ag-grid-docs/src/components/demos/DemosTabs.astro [hunk 1]
```diff
@@ -21,10 +21,10 @@ const { activeTab } = Astro.props;
                 />
             </a>
         </div>
-        <span>Performance</span>
+        <span class:list={[activeTab === 'complete' && styles.spanActive]}>Performance</span>
```
Comment (taktran) [inline documentation/ag-grid-docs/src/components/demos/DemosTabs.astro:24]: nit: The other way to conditionally apply the style is to use an object:

```suggestion
        <span class:list={[{ [styles.spanActive]: activeTab === 'complete' }]}>Performance</span>
```

**Review by taktran — APPROVED** [review]
👍🏼

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 2]
```diff
@@ -108,7 +139,22 @@
 .tabs span {
     font-weight: 600;
     display: flex;
-    justify-content: center;
     padding-top: $spacing-size-4;
-    color: var(--color-fg-primary);
+    color: color-mix(in srgb, var(--color-fg-primary), var(--color-text-tertiary) 100%);
+}
+
+#{$selector-darkmode} span {
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:146]: This line will affect ALL spans in darkmode?

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 3]
```diff
@@ -9,10 +9,6 @@
     height: 100%;
 }
 
-.showcaseItemTab {
-    box-shadow: var(--shadow-sm);
-}
-
 #{$selector-darkmode} .tabWrapper {
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:12]: Generally would prefer the parent selector approach for dark-mode stuff. E.g. 

Note the `&` after `#{$selector-darkmode}`. 

```
.tabwrapper {

  #{$selector-darkmode} & {
    // darkmode styles
  }
}

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 4]
```diff
@@ -72,28 +88,43 @@
...
         opacity: 1;
-        transition: all 0.3s ease-in-out;
+        transition: all 0.4s ease-in-out;
     }
 }
 
 .tabActive {
-    border: 2px solid #044b93;
+    outline: 2px solid #044b93;
     border-radius: var(--radius-lg);
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:104]: Add a new line between parent rules and nested selectors

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 5]
```diff
@@ -72,28 +88,43 @@
...
+    span {
+        color: #044b93;
+    }
+}
+
+#{$selector-darkmode} .tabActive {
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:110]: Would prefer the parent selector approach to darkmode stuff. See top comment for example.

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 6]
```diff
@@ -72,28 +88,43 @@
...
 }
 
 .tabs a div {
     display: flex;
-    align-items: center;
-    justify-content: center;
     padding-top: 12px;
 }
 
+.tabItem span {
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:119]: `.tabItem span` & `.tabItem:hover span` could be combined into nested styles for `.tabItem`

File: documentation/ag-grid-docs/src/components/demos/Demos.module.scss [hunk 7]
```diff
@@ -108,7 +139,22 @@
...
+
+.tabs .spanActive {
+    color: #044b93;
+}
+
+#{$selector-darkmode} .tabs .spanActive {
+    color: var(--color-link);
+}
+
+#{$selector-darkmode} .tabs span {
```
Comment (markdurrant) [inline documentation/ag-grid-docs/src/components/demos/Demos.module.scss:158]: Would prefer the parent selector approach to darkmode stuff. See top comment for example.

**Pull request merged**